specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2024-01-01-preview/AlertsManagement.json

```diff
@@ -0,0 +1,1598 @@
+          "readOnly": true,
+          "enum": [
+            "Application Insights",
+            "ActivityLog Administrative",
+            "ActivityLog Security",
+            "ActivityLog Recommendation",
+            "ActivityLog Policy",
+            "ActivityLog Autoscale",
+            "Log Analytics",
+            "Nagios",
+            "Platform",
+            "SCOM",
+            "ServiceHealth",
+            "SmartDetector",
+            "VM Insights",
+            "Zabbix",
+            "Resource Health"
+          ],
+          "x-ms-enum": {
+            "name": "MonitorService",
+            "modelAsString": true
+          }
+        },
+        "alertRule": {
+          "type": "string",
+          "description": "Rule(monitor) which fired alert instance. Depending on the monitor service,  this would be ARM id or name of the rule.",
+          "readOnly": true
+        },
+        "sourceCreatedId": {
+          "type": "string",
+          "description": "Unique Id created by monitor service for each alert instance. This could be used to track the issue at the monitor service, in case of Nagios, Zabbix, SCOM etc.",
+          "readOnly": true
+        },
+        "smartGroupId": {
+          "type": "string",
+          "description": "Unique Id of the smart group",
+          "readOnly": true
+        },
+        "smartGroupingReason": {
+          "type": "string",
+          "description": "Verbose reason describing the reason why this alert instance is added to a smart group",
+          "readOnly": true
+        },
+        "startDateTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Creation time(ISO-8601 format) of alert instance.",
+          "readOnly": true
+        },
+        "lastModifiedDateTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Last modification time(ISO-8601 format) of alert instance.",
+          "readOnly": true
+        },
+        "monitorConditionResolvedDateTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Resolved time(ISO-8601 format) of alert instance. This will be updated when monitor service resolves the alert instance because the rule condition is no longer met.",
+          "readOnly": true
+        },
+        "lastModifiedUserName": {
+          "type": "string",
+          "description": "User who last modified the alert, in case of monitor service updates user would be 'system', otherwise name of the user.",
+          "readOnly": true
+        },
+        "actionStatus": {
+          "$ref": "#/definitions/actionStatus"
+        },
+        "description": {
+          "type": "string",
+          "description": "Alert description."
+        },
+        "hasEnrichments": {
+          "type": "boolean",
+          "description": "Will be presented with the value true only if there are enrichments.",
```

Comment:
>Will be presented with the value true only if there are enrichment [](http://example.com/codeflow?start=26&length=66)

suggestion:
Present with the value true, only if there are enrichments. Default is false.